index.html

```diff
@@ -0,0 +1,66 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <header id="header">
+        <div class="flex-container">
+          <nav id="nav-bar">
+            <div id="logo-placeholder">
+              <a href="index.html">John Doe</a>
+            </div>
+            <div id="normal-button">
+              <img id="menu-button" src="icons/Icon - Menu.svg" alt="menu-button"/>
+            </div>
+          </nav>
+        </div>
+      </header>  
+      <ul id="headline">
+          <li>
+            <h1 id="primary-text">Hey There. I'm Eduardo</h1>
+          </li>
+          <li>
+            <h1 id="supporting-text">I'am a Software Developer</h1>
+          </li>
+          <li>
+            <h1 id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</h1>
+          </li>
+          <li>
+            <ul id="social-media">
+              <li>
+                <a href="https://github.com/evillaz">            
+                  <img id="logo1" src="icons/logo1.svg" alt="logo1" />
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/eduardo-villarreal-144a8925a/">
+                  <img id="logo2" src="icons/logo2.svg" alt="logo2" />
+                </a>
+              </li>
+              <li>
+                <a href="https://angel.co/u/eduardo-villarreal-3">
+                  <img id="logo3" src="icons/logo3.svg" alt="logo3" />
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/evillazz">
+                  <img id="logo4" src="icons/logo4.svg" alt="logo4" />
+                </a>
+              </li>
+              <li>
+                <a href="https://medium.com/@evillaz123">
+                  <img id="logo5" src="icons/logo5.svg" alt="logo5" />
```

Comment:
[OPTIONAL] I highly suggest that you make the `alt` attributes more descriptive because it will increase your website accessibility (When a user is using a screen reader it read the `alt` value whenever it encounters an `img` tag check [this](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) for more info)